docs/zh/dev/build.md

```
@@ -8,159 +8,76 @@
 
-### LLVM Libs
+## CMake
 
-- 20.1.5 <= llvm libs < 21
+使用如下的命令构建 clice
 
-由于 C++ 的语法太过复杂，自己编写一个新的 parser 是不现实的。clice 调用 clang 的 API 来 parse C++ 源文件获取 AST，这意味它需要链接 llvm/clang libs。另外由于 clice 使用了 clang 的私有头文件，这些私有头文件在 llvm 发布的 binary release 中是没有的，所以不能直接使用系统的 llvm package。
-
-如果你能找到系统的 llvm package 对应的 llvm commit，将该 commit 下的如下三个文件
+```shell
+$ cmake -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++
```

markdownlint-cli2 (0.18.1)

Dollar signs used before commands without showing output

(MD014, commands-show-output)